### DISCUSSION TITLE
新版动态页面夜间模式适配不完全

### DISCUSSION THREAD

**Discussion by Alexius66** [the1812/Bilibili-Evolved#4163] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/73510996/234276334-2a32e1f1-a419-4bf6-b950-0de133c9306f.png)

**Comment by the1812** [comment]
#3917

**Comment by tiger2005** [comment]
![image](https://user-images.githubusercontent.com/41613797/234570847-10000f6d-24b4-41d2-8d3d-e737496bb92e.png)
动态界面的广告横幅也没有适配，看起来有点眼瞎（

**Comment by tiger2005** [comment]
对 `dyn-upower-lottery__card` 类的 CSS 没有被覆盖，对应的是动态的充电栏。这个改一下 `dyn-reserve__card` 的 CSS 就行。
